src/functionHelper.js

```diff
@@ -18,9 +24,66 @@ module.exports = {
     };
   },
 
+  createExternalHandler(funOptions, options) {
+    let handlerContext = handlerCache[funOptions.handlerPath];
+
+    function handleFatal(error) {
+      handlerContext.inflight.forEach(id => messageCallbacks[id](error));
+      delete handlerCache[funOptions.handlerPath];
+    }
+
+    if (!handlerContext) {
+      debugLog(`Loading external handler... (${funOptions.handlerPath})`);
+
+      const helperPath = path.resolve(__dirname, 'ipcHelper.js');
+      const ipcProcess = fork(helperPath, [funOptions.handlerPath], {
+        env: JSON.parse(JSON.stringify(process.env)), // hacky workaround for undefined values
```

Comment:
What's the purpose of this? Cloning `process.env`? If so could you just use lodash `cloneDeep`?

Reply:
It appears to be a node bug where `undefined` values in process.env are passed to child processes as the literal string "undefined".  JSON.stringify/parse removes those values. Could use `_.omitBy(process.env, _.isUndefined)` instead.

Reply:
@encounter I think that's a better approach 👍

Reply:
@encounter Do you mind making this small update? I'll merge after.

Reply:
Done! Thanks.